algolia-config.json

```diff
@@ -0,0 +1,44 @@
+{
+  "index_name": "greatexpectations_trial_doc_try3",
```

Comment:
A bit of a nitpick, but could we make this name a little bit more "official"?

Reply:
@Shinnnyshinshin, what would you suggest is a better name for this?

Reply:
Let's do great_expectations_doc_index_v1